edb/server/main.py

```diff
@@ -56,6 +57,8 @@
 from . import protocol
 
 
+BYTES_OF_MEM_PER_CONN = 100 * 1024 * 1024 / 8  # 100Mb
```

Comment:
Why are you dividing by 8 here?  

Also, 100MB per backend seems a bit low, but, on the other hand, actual worker memory utilization depends greatly on the complexity and number of queries in prepared statements, so we will need to build tools to query or estimate actual memory utilization and have the pool retire connections that have crossed a certain memory utilization threshold.

Reply:
Oh the `8` is for `Mb` bytes to bits, but if 100MBytes is already too low ... I'll definitely remove the `/ 8` then, and perhaps we could start with 100MBytes per backend assumption? That's approximilately 10 backend connections per GB of memory, or 82 for 8GB.

Reply:
Oh.  Memory isn't usually measured in bits, so this got me confused.  Yes, we want at least 100MiB, but, like I said above, ideally this heuristic should be dynamic.